base/abstractarraymath.jl

```diff
@@ -84,8 +84,36 @@ function _dropdims(A::AbstractArray, dims::Dims)
     end
     reshape(A, d::typeof(_sub(axes(A), dims)))
 end
+
 _dropdims(A::AbstractArray, dim::Integer) = _dropdims(A, (Int(dim),))
 
+"""
+    dropdims(f, args...; dims, kwargs...)
+
+Compute reduction `f` over dimensions `dims` and drop those dimensions from the result.
+
+# Examples
+```jldoctest
+julia> a = [3.0  2.0  6.0  8.0
+            6.0  1.0  4.0  2.0
+            3.0  0.0  7.0  6.0];
```

Comment:
Not sure on expected style here... just wanted something short and clear

Also, using `Float`s not `Int`s since `Int` is different on 32 and 64 bit systems